### DISCUSSION TITLE
Renovate does not create prs when schedule is narrow

### DISCUSSION THREAD

**Discussion by alvinsjoy** [renovatebot/renovate#29695] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

Hi,
In my [renovate.json](https://github.com/alvinsjoy/PortfolioWebsite/blob/bfa0498599192a9076da25342e90e9c607506953/renovate.json), I have scheduled PR creation as `before 1am` but renovate is not creating PRs even after it is listed on the [dependency dashboard issue](https://github.com/alvinsjoy/PortfolioWebsite/issues/70) for two days.
See the image below for better understanding.
![image](https://github.com/renovatebot/renovate/assets/89687635/dd1416e9-908e-46ee-93c9-f1653b95391f)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
Please read https://docs.renovatebot.com/key-concepts/scheduling/

Especially:

> Be sure to schedule enough time for Renovate to process your repository. Do not set schedules like "Run Renovate for an hour each Sunday" as you will run into problems.

"Before 1am" is too narrow of a time because the hosted bot typically runs every 3-4 hours unless there's a webhook which triggers a run sooner. Use "Before 4am" instead.

**Reply by alvinsjoy** [reply]
Thanks I'll try that.
